projects/components/src/button/button.component.ts

```diff
@@ -59,7 +59,7 @@ export class ButtonComponent {
   public type: ButtonType = ButtonType.Button;
 
   @Input()
-  public role: ButtonRole = ButtonRole.Secondary;
+  public variant: ButtonVariant = ButtonVariant.Secondary;
```

Comment:
In addition to this, I think we should set an aria role attribute on the root element

Reply:
It's there on line 15. I searched for all button elements in the code. I think there were only like 4 or 5 because we use our own wrappers.

Reply:
Sorry, i think i commented before the latest commit.

Reply:
Oh, hah. Yeah, I tried to break up the commits for easier reviewing.